packages/arb-token-bridge-ui/src/token-bridge-sdk/EthWithdrawalStarter.ts

```diff
@@ -45,19 +46,29 @@ export class EthWithdrawalStarter extends BridgeTransferStarter {
     })
   }
 
-  public async transfer({ amount, signer }: TransferProps) {
+  public async transfer({ amount, signer, destinationAddress }: TransferProps) {
```

Comment:
for some reason it just doesn't work for me, i tried twice :/ 

<img width="1037" alt="image" src="https://github.com/user-attachments/assets/4ddbbbdd-3f44-4e0f-adf0-75b567faf38d">

Reply:
it works today, could be an issue due to gas spike